### PR TITLE
Disable starting in the overview

### PR DESCRIPTION
**Pull request by vanvugt**
We don't need it when the GNOME dash is being replaced.
([LP: #1940925](https://bugs.launchpad.net/bugs/1940925))

**Requires:** https://salsa.debian.org/gnome-team/gnome-shell/-/merge_requests/52 so this really is intended for ubuntu-dock only.

CC @3v1n0

### CODE REVIEW

**Comment by vanvugt** [comment]
I'm not sure if this is your preferred location to set the flag so feel free to move it... it's trivial so you don't need my permission.

File: extension.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ function conditionallyenabledock() {
 
     // enable or disable dock depending on dock status and to_enable state
     if (to_enable && !dockManager) {
+        if (Main.layoutManager.startInOverview !== undefined)
```
[the remark this answers is not in the document]
Comment (vanvugt) [reply extension.js:52]: I have a new plan that will eventually require the `if` statement is used...

[the remark this answers is not in the document]
Comment (vanvugt) [reply extension.js]: Changed my mind again. The `if` is gone.

[the remark this answers is not in the document]
Comment (gunnarhj) [reply extension.js]: Hey, not a one-liner any longer... :wink: Leaving it to someone with write access to this repo to update the `gnome-shell-extension-ubuntu-dock` package with the additional stuff.

**Comment by vanvugt** [comment]
This also resolves https://bugs.launchpad.net/bugs/1942843

**Comment by gunnarhj** [comment]
Uploaded to impish as `gnome-shell-extension-ubuntu-dock 70~ubuntu2` and `70~ubuntu3`.

**Pull request merged**